components/config-mgmt-service/backend/elastic/custom_search_aggs_bucket_date_range.go

```diff
@@ -0,0 +1,267 @@
+package elastic
```

Comment:
This is the file that was copied from olivere to add the "missing" option. It was copied from https://github.com/olivere/elastic/blob/release-branch.v6.